app/views/user/shares/_form.html.erb

```diff
@@ -1,13 +1,19 @@
 <%= bootstrap_form_with(url: user_shares_path, layout: :horizontal, scope: 'share', local: true,) do |f| %>
-<h5>Type name of organisation</h5>
-  <div>
+<h5 class="font-weight-bold mb-4">Feijoa-registered recipient</h5>
+  <%# <div>
     Sharing birth record of <%= record.full_name %>
-  </div>
+  <%# </div> %>
   <%= f.hidden_field :document_id, value: record.id %>
   <%= f.hidden_field :recipient_id, id: 'recipient_id', value: nil %>
   <div >
     <%= f.text_field :organisation, id: "org-query", label_as_placeholder: true, label: 'Organisation' %>
   </div>
+  <hr />
+  <h5 class="font-weight-bold mt-4 mb-4">or email</h5>
```

Comment:
This is something that might be legitimate to comment out, since it's a piece of UI for a feature that doesn't exist.